client/src/main/scala/org/apache/celeborn/client/ChangePartitionManager.scala

```diff
@@ -253,6 +253,7 @@ class ChangePartitionManager(
         .keySet()
         .asScala
         .filter(w => !lifecycleManager.workerStatusTracker.blacklist.keySet().contains(w))
```

Comment:
It's nice if encapsulate these filter logic into workerStatusTracker

Reply:
Done